lib/integrations/hip_chat.rb

```diff
@@ -51,5 +51,13 @@ def message_color
 
       color_hash[event_name]
     end
+
+    def run_href
+      "<a href=\"#{payload[:frontend_url]}\">Run ##{run[:id]}#{run_description}</a>"
```

Comment:
I don't really mind either way. I just picked the default as text because it will always look fine no matter what the integrations support.